sql/item_subselect.h

```diff
@@ -360,12 +360,12 @@ class Item_maxmin_subselect :public Item_singlerow_subselect
 public:
   Item_maxmin_subselect(THD *thd, Item_subselect *parent,
 			st_select_lex *select_lex, bool max);
-  virtual void print(String *str, enum_query_type query_type);
-  void cleanup();
+  virtual void print(String *str, enum_query_type query_type) override;
```

Comment:
I’d be consistent and remove `virtual` from anything that is declared `override`.

Reply:
Agreed, will do as part of this patch.